src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -147,45 +145,36 @@ We then pass this `FlutterEngine` into a
 {% tabs "darwin-framework" %}
 {% tab "SwiftUI" %}
 
-The following example shows a generic `ContentView` with a
-`Button` hooked to present a [`FlutterViewController`][].
-The `FlutterViewController` constructor takes the pre-warmed 
-`FlutterEngine` as an argument. `FlutterEngine` is passed in 
-as an `EnvironmentObject` via `flutterDependencies`.
+The following example shows a generic `ContentView` with a `NavigationLink` hooked to a flutter screen. 
+We firstly create a `FlutterViewControllerRepresentable` to represent the `FlutterViewController`. 
+The `FlutterViewController` constructor takes the pre-warmed `FlutterEngine` as an argument, which is injected via environment. 
```

Comment:
```suggestion
The following example shows a generic `ContentView` with a
`NavigationLink` hooked to a flutter screen. 
First, create a `FlutterViewControllerRepresentable` to represent the
`FlutterViewController`. The `FlutterViewController` constructor
takes the pre-warmed `FlutterEngine` as an argument, which is injected through
the `environment` property. 
```

Reply:
Also, consider linking `NavigationLink` to https://developer.apple.com/documentation/swiftui/navigationlink

Reply:
done and done